eclipse-collections/pom.xml

```diff
@@ -35,23 +35,50 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-primitive-collections-base</artifactId>
+            <version>${project.version}</version>
+        </dependency>
         <dependency>
             <groupId>org.eclipse.collections</groupId>
             <artifactId>eclipse-collections</artifactId>
-            <version>9.1.0</version>
+            <version>${version.eclipse-collections}</version>
         </dependency>
     </dependencies>
 
     <build>
         <plugins>
-          <plugin>
-            <groupId>com.google.code.maven-replacer-plugin</groupId>
-            <artifactId>replacer</artifactId>
-          </plugin>
-	  <plugin>
-            <groupId>org.moditect</groupId>
-            <artifactId>moditect-maven-plugin</artifactId>
-	  </plugin>
+            <plugin>
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>io.timeandspace</groupId>
+                <artifactId>jpsg-maven-plugin</artifactId>
+                <version>1.3</version>
+                <executions>
+                    <execution>
+                        <id>jpsg</id>
+                        <goals>
+                            <goal>jpsg</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>com/fasterxml/jackson/**/failing/*.java</exclude>
```

Comment:
Is this still required? I assume this was for #54

Reply:
I copied this from `jackson-datatype-collections/pom.xml`. It is a general config, apparently, for local development, where failing tests could be temporarily moved to `...failing` package. Not related to #54.

However, this config is inherited from `jackson-datatype-collections/pom.xml` in `eclipse-collections.xml` anyway, so I removed it here.

I've also found that guava, pcollections, and primitive-collections-base lacked the spec of maven-surefire-plugin and the tests were not run during `install` goal. Added maven-surefire-plugin to the config of these modules.